index.html

```diff
@@ -533,12 +533,12 @@ <h4 id="editor-linux">Linux</h4>
     <p>
       Regardless of how you choose to install it,
       <strong>please make sure you install Python version 3.x</strong>
-      (e.g., 3.4 is fine).
+      (e.g., 3.6 is fine).
     </p>
 
     <p>
-      We will teach Python using the IPython notebook, a programming environment
-      that runs in a web browser. For this to work you will need a reasonably
+      We will teach Python using the <a href="https://jupyter.org/">Jupyter notebook</a>,
+      a programming environment that runs in a web browser. For this to work you will need a reasonably
       up-to-date browser. The current versions of the Chrome, Safari and
       Firefox browsers are all
       <a href="http://ipython.org/ipython-doc/2/install/install.html#browser-compatibility">supported</a>
```

Comment:
This link is going to outdated IPython documentation, however, http://jupyter.readthedocs.io/ does not seem to have an item on browser compatibility anymore.